README.md

```diff
@@ -141,6 +141,7 @@ sun_tools  *        *     *     *     *    *
 
 * [`batch-pacapt`](https://github.com/Grenadingue/batch-pacapt): An Arch's pacman-like package manager for Windows
 * [`node-pacapt`](https://github.com/Grenadingue/node-pacapt): A node.js wrapper of pacapt + batch-pacapt
+* [`pacaptr`](https://github.com/rami3l/pacaptr): A Rust port of pacapt with decent `homebrew`/`choco`/`dpkg` support and more
```

Comment:
Can we leave a note somewhere in the README file, to emphasize that we want to have a better port/future for the project? A link to the our original discussion in https://github.com/icy/pacapt/issues/126 is also great. Thanks a lot.

Reply:
Yup.
Initially I wanted to catch attention, but in order to do that, the changes that have to be made to README might look a bit more “aggressive”...
And I am asking myself, “Am I the right one to say something about the future of this project?”

Reply:
We can do that together ;) I have updated the PR, I hope that's not strong and/or too much aggressive; feel free to improve that commit. 

`pacapt` is now part of a few package systems (e.g https://packages.ubuntu.com/search?keywords=pacapt), and we don't like to confuse / break everything. 

Your project is very young, we can't say much now. But as discussed in #126 it's better future, so why don't we can wait a bit ;)

Reply:
That looks great!

I would like to do some further modifications, but sadly not now, as I have to return to my courses on Monday…

Please be patient, and thanks a lot for your support, as always :)

Reply:
It's better now. Can I merge the PR?

Reply:
Yes of course.